Tools-Override/tests.targets

```diff
@@ -211,8 +211,54 @@
       <TestCommandLines Include="cd native"/>
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPRunnerFolder)'==''" Text="Please specify UAPRunnerFolder" />
```

Comment:
This is going to cause everyone that just tries to build tests for uap to specify UAPRunnerFolder and UAPLauncherFolder that doesn't seem like a good idea to me.